keystorage/hashicorp/build.gradle

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2019 ConsenSys AG.
+ *
+ * The source code is provided to licensees of PegaSys Plus for their convenience and internal
+ * business use. These files may not be copied, translated, and/or distributed without the express
+ * written permission of an authorized signatory for ConsenSys AG.
+ */
+
+apply plugin: 'java-library'
+apply plugin: 'java-test-fixtures'
+
+jar {
+  archiveBaseName = "hashicorp-vault"
```

Comment:
Think this should match the module name, so keystore-hashicorp instead?

Reply:
done

Reply:
still looks the module name is "hashicorp-vault" ?